odata-parser.d.ts

```diff
@@ -27,9 +27,16 @@ interface GenericPropertyPath<T = any> {
 	name: string;
 	property?: T;
 }
-export type PropertyPath = GenericPropertyPath<PropertyPath>;
-export interface OrderByPropertyPath
-	extends GenericPropertyPath<OrderByPropertyPath> {
+export type PropertyPath = GenericPropertyPath<PropertyPath> & {
+	key?:
+		| BindReference
+		| {
+				[resourceName: string]: BindReference;
+		  };
+	count?: true;
+	options?: { $filter: FilterOption };
+};
+export interface OrderByPropertyPath extends GenericPropertyPath<PropertyPath> {
```

Comment:
Children of the OrderByPropertyPath node do no have `asc/desc`, so I switched this to `PropertyPath` as per the source:
```
SortProperty =
	property:PropertyPath
	order:(
		boundary
		@(
			'asc'
		/	'desc'
		)
	/	'' { return 'desc' }
	)
	{
		property.order = order;
		return property
	}
```